extensions/cornerstone-3d/src/init.js

```diff
@@ -36,6 +37,9 @@ export default async function init({
   await cs3DInit();
   initCornerstoneTools();
 
+  // Don't use cursors in viewports
+  Settings.getRuntimeSettings().set('useCursors', false);
```

Comment:
Shouldn't we set this based on extension configuration?

...aside: are we even using extension configuration. How do we use it with the current ohif-cli dev flow?